timeless/views.py

```diff
@@ -266,9 +262,15 @@ class UpdateView(GenericView):
     """ Base view for updating objects"""
 
 
-class DeleteView(GenericView):
+class DeleteView(SuccessRedirectMixin, SingleObjectMixin, GenericView):
     """ Base view for deleting objects """
```

Comment:
@vryazanov Can you add more details to this docstring? Maybe point the reader to a place where it is used, to see it in action. I hate that apparently people don't write docstrings/javadocs in their pythin code.